app/views/catalog/_show_related_works_default.html.erb

```diff
@@ -2,11 +2,13 @@
 <div class="show-included-works">
   <% if display_values.any? %>
     <h2><%= t('trln_argon.show.heading.related_works') %></h2>
-      <dl class="dl-invert">
+      <div>
       <% display_values.each do |value| %>
-          <dt><%= value[:label] %></dt>
-          <%= value[:value] %>
+          <h3><%= value[:label] %></h3>
+          <ul>
+          	<li><%= value[:value] %></li>
```

Comment:
One problem is that the way this is done you only get a single li even if there are many metadata values. It would be better if you used an li for each metadata value.

Reply:
There are likely to be several places where changes are needed to make this happen. If I have time later today I can try to point out where you'll need to make adjustments.